code/mob/living/carbon/human/procs/Life.dm

```diff
@@ -478,6 +478,9 @@
 
 #if ASS_JAM //Oh neat apparently this has to do with cool maptext for your health, very neat. plz comment cool things like this so I know what all is on assjam!
 	src.UpdateDamage()
+#else
+	if(istype(src, /mob/living/carbon/human/dummy)) //do it for target dummies all the time
+		src.UpdateDamage() 
```

Comment:
I'd rather this just have

```
/mob/living/carbon/human/dummy/Life(datum/controller/process/mobs/parent)
  if (..(parent))
    return 1
  src.UpdateDamage()
```

You could put this bit right after your `/mob/living/carbon/human/dummy/UpdateDamage()` in `update_icon.dm` so it wouldn't double call on Ass Day.